yb-voyager/cmd/analyzeSchema.go

```diff
@@ -259,6 +260,29 @@ func reportSummary() {
 	}
 }
 
+func addSummaryDetailsForIndexes() {
+	var indexesInfo []utils.IndexInfo
+	found, err := metaDB.GetJsonObject(nil, metadb.INDEXES_INFO_KEY, &indexesInfo)
+	if err != nil {
+		utils.ErrExit("analyze schema report summary: load indexes info: %s", err)
+	}
+	if found {
+		indexesPresent := lo.Keys(summaryMap["INDEX"].objSet)
+		unexportedIdxsMsg := "Indexes which are neither exported by yb-voyager as they are unsupported in YB and needs to be handled manually:\n"
+		unexportedIdxsPresent := 0
+		for _, indexInfo := range indexesInfo {
+			exportedIdxName := indexInfo.TableName + "_" + strings.Join(indexInfo.Columns, "_")
```

Comment:
I think for Oracle it might be fine, but ideally, if there are multiple columns in the `indexInfo.Columns`, we should iterate all the columns and checked individual index name in the list right?